src/azure-cli/azure/cli/command_modules/acr/task.py

```diff
@@ -401,8 +404,11 @@ def acr_task_update(cmd,  # pylint: disable=too-many-locals, too-many-statements
 
     step = task.step
     branch = None
+
     if context_path is None:
```

Comment:
Could you add some comments here for the logic?

Reply:
done